### DISCUSSION TITLE
Dashy - icons config/location

### DISCUSSION THREAD

**Discussion by tomasenskede** [tteck/Proxmox#2842] [General]
I've set up Dashy using the script, and it was quick and straightforward as usual with these scripts. However, I'm facing an issue with configuring the use of icons. Does docker-compose play a role in this? Also, how can I redeploy if I've added volumes to it? Or is there a better way to configure the setup than what the script generated?

**Comment by tteck** [comment]
https://dashy.to/docs

docker-compose? volumes?

**Reply by tomasenskede** [reply]
Indeed, this is the correct way to do it... However, when using the script (https://tteck.github.io/Proxmox/) for deployment, I'm unable to locate Docker to redeploy with the updated docker-compose. Any suggestions on how to redeploy the Dashy container?

**Reply by tteck** [reply]
This does **NOT** use Docker.

**Reply by tomasenskede** [reply]
hmm, how can I get the icons/logo then into Dashy?

**Reply by tteck** [reply]
`conf.yml`

example: https://github.com/Lissy93/dashy/blob/master/user-data/conf.yml

**Reply by tomasenskede** [reply]
so I should link to external icon/logo? 

icon: https://i.ibb.co/qWWpD0v/astro-dab-128.png

... why cant I use local images?

**Reply by tteck** [reply]
You can
> An icon, either as a font-awesome, simple-icon or mdi identifier, emoji, favicon, generative or the URL/ path to a local or remote icon asset

**Reply by tomasenskede** [reply]
How can I specify the location for storing images/logos/icons? Using volumes in Docker Compose would be the approach for a Docker container, but how do I define the storage location in this setup?

**Reply by tteck** [reply]
![Screenshot 2024-04-23 8 22 32 AM](https://github.com/tteck/Proxmox/assets/21342670/d2a82ba5-2bb5-4f88-9fc4-d54fbb267b11)


👨🏻‍🦱 My focus is on developing application LXC scripts, and maintaining said scripts. I do not specialize in, nor do I have the time to provide support for operational issues. Hopefully someone will come along and assist you further.

**Reply by tomasenskede** [reply]
yes, i know this setings... but when I deploy Dashy on docker I can specify the location of imates like this;
![image](https://github.com/tteck/Proxmox/assets/32595351/f5435bfe-dc1c-4d54-a38d-b07a33d0a16b)
How can I get this settings in the setup?

**Reply by tteck** [reply]
This installation is in a **Linux container**, _some_ Linux familiarity is recommended.

To illustrate, if you were to place all your icons, such as astro-dab-128.png, within the dashy directory, the file path would simply be `astro-dab-128.png`.

**Reply by tomasenskede** [reply]
should i store my images in this folder /opt/dashy/ or?
